### PR TITLE
WPAPSK CPU formats: Remove dependency on OpenSSL for 802.11w support

### PR DESCRIPTION
**Pull request by magnumripper**
Use the same simple CMAC-AES-128 implementation we have in OpenCL.

### CODE REVIEW

**Comment by magnumripper** [comment]
I should have fixed this back in 2017 when I added the CMAC code for OpenCL. As expected, I see no change in performance - the AES-CMAC shouldn't be significant.

File: src/cmac_plug.c [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+ * WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ * MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+ * ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ * WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+ * ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+ * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+ */
+
+#include "cmac.h"
+#include "common.h"
```
Comment (solardiz) [inline src/cmac_plug.c:22]: What do we use from `common.h` here?

**Review by solardiz — APPROVED** [review]
I didn't review the CMAC code itself, but if the bots are happy it should be right.

File: src/cmac_plug.c (hunk 1 above)
Comment (magnumripper) [reply src/cmac_plug.c:22]: Just the MIN() macro.

**Pull request merged**